securedrop/alembic/versions/2d0ce3ee5bdc_added_passphrase_hash_column_to_.py

```diff
@@ -0,0 +1,53 @@
+"""added passphrase_hash column to journalists table
+
+Revision ID: 2d0ce3ee5bdc
+Revises: faac8092c123
+Create Date: 2018-06-08 15:08:37.718268
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2d0ce3ee5bdc'
+down_revision = 'faac8092c123'
```

Comment:
test CI job is failing here as `down_revision` needs to be updated to `fccf57ceef02` after merge of #3619